### PR TITLE
Include test utils in NPM package

### PR DESCRIPTION
**Pull request by andreivladbrg**
Same rationale as [here](https://github.com/PaulRBerg/prb-math/pull/211)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,6 +32,10 @@
     "solidity"
   ],
   "license": "MIT",
+  "peerDependencies": {
+    "@forge/std": "github:foundry-rs/forge-std#e8a047e3f40f13fa37af6fe14e6e06283d9a060e",
+    "@prb/test": "0.6.4"
+  },
```
Comment (PaulRBerg) [inline package.json:38]: Should be removed

**Review by PaulRBerg — CHANGES_REQUESTED** [review]
thanks

File: package.json (hunk 1 above)
Comment (andreivladbrg) [reply package.json:38]: changed your mind about this? 😅

https://github.com/PaulRBerg/prb-math/pull/206#discussion_r1375931024

**Review by PaulRBerg — APPROVED** [review]
LGTM now

**Pull request merged**